protocol/src/main/java/org/fidoalliance/fdo/protocol/db/FdoSimCommandOwnerModule.java

```diff
@@ -0,0 +1,283 @@
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fidoalliance.fdo.protocol.db;
+
+import java.io.IOException;
+import java.util.Map;
+import org.fidoalliance.fdo.protocol.LoggerService;
+import org.fidoalliance.fdo.protocol.Mapper;
+import org.fidoalliance.fdo.protocol.dispatch.ServiceInfoModule;
+import org.fidoalliance.fdo.protocol.dispatch.ServiceInfoSendFunction;
+import org.fidoalliance.fdo.protocol.message.AnyType;
+import org.fidoalliance.fdo.protocol.message.DevModList;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoDocument;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoKeyValuePair;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoModuleState;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoQueue;
+import org.fidoalliance.fdo.protocol.serviceinfo.DevMod;
+import org.fidoalliance.fdo.protocol.serviceinfo.FdoSys;
+
+public class FdoSimCommandOwnerModule implements ServiceInfoModule {
+
+
+  public static final String MODULE_NAME = "fdo.command";
```

Comment:
fdo.command feels a little awkward for modules like command.

`fdo.command:command `

Can it be renamed to fsim or something simpler ? Just a suggestion.